### PR TITLE
Index example code

### PR DESCRIPTION
**Pull request by shinyou21**

### CODE REVIEW

File: Index/src/DataIndex.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	{
+	System.out.println("hello");
+		//List<Document> res;
+		String CONNECTION = "mongodb+srv://m001-student:student123#@sandbox-trhqa.mongodb.net/test";
+		String city = "Texas";
+		List<Document> filteredDocuments = GetShippingByCity(CONNECTION,city);
+	
+	}
+	
+	public static List<Document> GetShippingByCity(String connectionString, String city)
```
Comment (Raghav2018) [inline Index/src/DataIndex.java]: You could add a unit test to this method

**Comment by shinyou21** [comment]
@Raghav2018 made changes to the code as well as added unit test case.
Please review the same

File: Index/testsrc/DataIndexTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		public void checkIndexBefore_emptyValue_ThrowsException() 
+		{
+			DataIndex d_index = new DataIndex();	
+			List<Document> filteredDocuments = d_index.GetShippingByCityWithoutIndex(connectionString,"");
+			
+		}
+		
+		
+		@Test
+		public void checkIndexSizeAfter()
```
Comment (Raghav2018) [inline Index/testsrc/DataIndexTest.java]: Naming convention for unit tests - MethodYouAreTesting_ScenarioYouAreTesting_ExpectedOutput(params)

**Comment by Raghav2018** [comment]
LGTM

**Pull request merged**